website/case-studies/flint-cloud.md

```diff
@@ -0,0 +1,66 @@
+- Thus, in deploying the FLINT on remote cloud resources the main objective is to increase ease of use and reduce the dependency on local systems, configurations, etc 
+- Developers experienced with the FLINTcloud framework will be able to support existing users with CI/CD pipelines, simplify model development and deployment, and leverage auto-scaling for large FLINT runs.
+- Moja global does not offer FLINT as a Saas on a commercial level, however, through the FLINT Cloud project, they offer deployment templates and workflow training. In January 2022, the organization received a generous credit grant from Microsoft to develop templates for users that wish to deploy tools on [Azure cloud](https://moja.global/moja-global-wins-azure-cloud-grant/)
+
+## What is supported currently?
+Currently, the users can run the FLINT and GCBM examples on local and production environments
+
+- **Local environment** 
+  - The FLINT and GCBM example docker images can be pulled and built from the Github Container Registry. 
+  - The steps to [locally](https://github.com/moja-global/FLINT.Cloud/tree/master/kubernetes/local) deploy the FLINT and GCBM examples on Kubernetes are available 
+ 
+- **Production environment** : 
+ - Using Terraform, an IaaS, deployments can be made on Azure and GCP.
+ - Steps to deploy FLINT and GCBM examples on the [Azure Kubernetes Service](https://github.com/moja-global/FLINT.Cloud/tree/master/kubernetes/azure) are available
+
+### FLINT Example 
+The [FLINT Example](https://github.com/moja-global/FLINT.Cloud/tree/master/local/rest_api_flint.example) demonstrates the Point and the Rothc modules. 
+
+The **Point module** is a simple test module, incorporated to present the concepts of the FLINT such as [Pools, Fluxes, Events and Conservation of mass](https://docs.moja.global/en/master/FLINT/UnderstandingFLINT/MojaFlint/index.html)
+
+More about the [**RothC module**](https://community.moja.global/case-studies/introduction-to-flint-modules#rothc-soil-carbon-module) can be found in the case study on flint modules
+
+
+### GCBM Example
+
+The [GCBM example](https://github.com/moja-global/FLINT.Cloud/tree/master/local/rest_api_gcbm) enables users to understand the entire GCBM simulation lifecycle. A GCBM simulation can be created in a few minutes, with no pre-requisites, apart from having about 5.95 GB for the Docker image. There are endpoints to create a new simulation, upload inputs, start the simulation, and obtain the outputs.
+
+Earlier, the inputs required to run the GCBM simulation were: configuration JSON files, input database and a folder containing the disturbances files (optional), classifier TIFF and JSON files
+Currently, only the disturbance and classifier TIFF files, along with input database are needed. A default configuration is used to run the simulation
+
+A future scope is to enable the users to edit configuration files and choose the modules that will be required
```

Comment:
```suggestion
In future, we plan to provide support so that users can edit configuration files and choose which modules will be required for the simulation.
```